echemdb/data/cv/entry.py

```diff
@@ -100,6 +100,35 @@ def __getitem__(self, name):
         """
         return Descriptor(self.package.descriptor)[name]
 
+    def citation(self):
```

Comment:
Shouldn't this be a `@property`?

Reply:
Sure at the moment, it is indeed a property.
I was wondering if we want to provide a bit more flexibility to the output style. for example to provide an option to enable/disable the title or to print all authors. until then I guess it can be a property.

Reply:
```suggestion
    @property
    def citation:
```